cylc/flow/remote.py

```diff
@@ -223,15 +222,14 @@ def construct_rsync_over_ssh_cmd(
 
 
 def construct_ssh_cmd(
-        raw_cmd, user=None, host=None, forward_x11=False, stdin=False,
```

Comment:
The `user` appears to be calculated in runtime, with no need to pass a `user` parameter. This `user` parameter is never used within `construct_ssh_cmd`, which can be misleading to a user/caller of the API.

Reply:
I think this was associated with the "task owner" concept we used to have (run a task job as user@host) but recently removed (now use ssh config instead if you need to run as a different user on host).